advanced_source/privateuseone.rst

```diff
@@ -0,0 +1,310 @@
+Facilitating New Backend Integration by PrivateUse1
+===================================================
+
+In this tutorial we will walk through some necessary steps to integrate a new backend
+living outside ``pytorch/pytorch`` repo by ``PrivateUse1``. Note that this tutorial assumes that
+you already have a basic understanding of PyTorch.
```

Comment:
```suggestion
you are and advanced user of PyTorch.
```
It feels like this tutorial requires not only basic understanding but more like advanced knowledge.

Reply:
Yes, advanced user needed